### DISCUSSION TITLE
Calling yup setLocale

### DISCUSSION THREAD

**Discussion by abduraufsherkulov** [i18next/next-i18next#1585] [Q&A] [answered]
I am using yup + react-hook-form and I am setting global [setLocale](https://github.com/jquense/yup#using-a-custom-locale-dictionary). I was doing so and including it in `_app.tsx`. However with next-i18next, I don't know how to call it since `serverSideTranslations` can only be called in pages. In other words, how can I use i18n instance outside pages. It was possible in react-i18next.

**Comment by isaachinman** [comment, marked as answer]
You can use:

```tsx
import { i18n } from 'next-i18next'

const translation = i18n.t('my-key')
```
